app/controllers/Checkout.scala

```diff
@@ -309,6 +308,7 @@ object Checkout extends Controller with LazyLogging with CatalogProvider {
       val eventualMaybeSubscription = tpBackend.subscriptionService.get[com.gu.memsub.subsv2.SubscriptionPlan.ContentSubscription](Name(subsName))
       eventualMaybeSubscription.map { maybeSub =>
         maybeSub.map { sub =>
+          if (tpBackend.environmentName == "PROD") Tip.verify()
```

Comment:
@pvighi suggested that I moved the verification step here, as this means that we will also catch a situation where we have broken the logic which displays the thankyou page